src/LoginDialog/OidcUtilities.ts

```diff
@@ -25,9 +25,9 @@ export const generateOidcLoginRedirectUrl = (baseRedirectUrl: string, cId: strin
         'scope=openid+email+profile'
     ].join('&');
     
-    return `${baseRedirectUrl}?${oidcParameters}`
+    return `${baseRedirectUrl}&${oidcParameters}`
```

Comment:
does this corner case even makes sense: 

what if baseRedirectUrl ends with `&` ?

Reply:
actually maybe it doesn't because the URL will still be valid, right?